README.md

```diff
@@ -67,10 +67,18 @@ yarn add react-native-tableview
 
 > If using CocoaPods or React Native version >= 0.60.0
 
+Intel macOS
+
 ```bash
 cd ios && pod install && cd ..
 ```
 
+M1 macOS
+
+```bash
+cd ios && arch -x86_64 pod install && cd ..
```

Comment:
With the latest version of cocoapods from `brew` we don't need this alias anymore.

Reply:
Thanks for the heads up! I'll update my version